pkg/app/instances/docker.go

```diff
@@ -82,7 +82,7 @@ func (m *DockerInstanceManager) CreateHost(zone string, _ *apiv1.CreateHostReque
 		},
 	}
 	hostConfig := &container.HostConfig{
-		Privileged:      true,
+		Privileged: true,
```

Comment:
nit: amend this change to the previous commit, having 2 commits on this PR only.

Reply:
Amended formatting change into previous one, Done.